### PR TITLE
Added assistance_rules layout

### PR DESCRIPTION
**Pull request by fedescarpa**
:warning: Please, see https://github.com/mumuki/mumukit-assistant/pull/8 first :warning:

ezgif com-gif-maker

### CODE REVIEW

File: app/helpers/assistance_box_helper.rb [hunk 1]
```diff
@@ -6,4 +6,8 @@ def assistance_box(assignment)
       </div>}.html_safe
     end
   end
+
+  def assistance_rules_passed?(assignment)
+    Mumukit::Assistant. parse(assignment.exercise.assistance_rules).rules.any? { |rule| rule.matches? assignment }
```
Comment (luchotc) [inline app/helpers/assistance_box_helper.rb:11]: Let's please move this logic to mumukit assistant if possible.
Also we could rename it to `should_display_assistance_box?`

**Review by luchotc — COMMENTED** [review]
Everything else looks good. I'll wait for @Gustrucco and @rociogonzalezt feedback

**Comment by luchotc** [comment]
Nice! Visually we've agreed with @rociogonzalezt and @Gustrucco that we should display them as opened until we improve that UI.
@fedescarpa please let's create a method `should_display_assistance_box?` in `assignment` that can be defined as: `tips.present?`.
Also note that assistance_box is already using that, so we can remove that `if` from there and move it to the layout using that method :smile:

Luckily there's no need to extract anything to mumukit-assistant :relieved:

File: app/helpers/assistance_box_helper.rb (hunk 1 above)
Comment (flbulgarelli) [reply app/helpers/assistance_box_helper.rb:11]: Also please remember there is already an `assistant` property with the rules parsed: 

https://github.com/mumuki/mumuki-domain/blob/dc34611ace2ece06a66590d0ae017f083b84ff27/app/models/concerns/assistable.rb#L10

Thus, we should write this code as `assistant.rules.any?  ...`

Comment (luchotc) [reply app/helpers/assistance_box_helper.rb:11]: Sorry, maybe my last comment wasn't clear enough. There's no necessary for this code neither, cause using `tips.present?` in assignment is sufficient.

**Comment by luchotc** [comment]
@fedescarpa I've solved it myself. Please do a final revision and manual test and merge it :smile: 

I've added an extra condition to be consistent with the previous behavior which had that condition in the layout.

**Pull request merged**